doc/z-chap10.xml

```diff
@@ -4,9 +4,12 @@
   </Heading>
 
   This chapter describes the functions in &Semigroups; for dealing with free
-  inverse semigroups and free bands. This part of the manual and the functions
-  described herein were written by Julius Jonu&#353;as.<P/>
-
+  inverse semigroups and free bands. Sections 10.1 to 10.5 in this chapter,
+  and the functions described therein, were written by Julius Jonu&#353;as.
+  Section 10.6 was written by Maria Tsalakou and Murray Whyte, and Section
+  10.7 was written by Luke Elliott.
```

Comment:
If the the relevant sections in this chapter were labelled (i.e. `<Section Label="blah">`), then these references could be proper cross-references, by writing `Section <Ref Sect="blah" Style="Number"/>`. This could be particularly handy if this chapter is ever reordered (either internally, or it becomes Section 11, say), as that means the references will update automatically somewhat, too.

Reply:
That's a great idea @wilfwilson, thanks, I'll push that now.